### PR TITLE
feat(website): move terms of use and privacy policy to subpages

### LINKED ISSUE #1844: make terms of use and privacy policy dynamic
Currently we have 2 pdfs on siasky.net that are linked in the footer:

- https://siasky.net/terms.pdf
- https://siasky.net/privacy.pdf

We need to move them to be subpages and make the portal domain dynamic because the pdf mentions siasky.net and once we deploy new portals these terms and privacy policy need to reflect the domain that the portal operates on. For domain name we want to use `PORTAL_DOMAIN` environment variable.

### PR DESCRIPTION
**Pull request by meeh0w**
## Example for Visual Changes
* [Privacy policy](https://user-images.githubusercontent.com/3853033/157195668-29b6cc25-6700-4eb2-8847-42d8a2d47aae.png)
* [Terms of use](https://user-images.githubusercontent.com/3853033/157195683-a3685274-fdaa-4b55-978f-13297dbada2c.png)

## Issues Closed
Closes #1844

### CODE REVIEW

File: packages/website/src/pages/terms.js [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+        including, without limitation, by generating recurring traffic to such website through the conduct of tests
+        using our Services; modify, publish, transmit, transfer or sell, reproduce, create derivative works from,
+        distribute, perform, display or in any way exploit any of the content of the Website in whole or in part, except
+        as expressly permitted; or use automated tools to operate the Services on the Website (scripts, etc.).
+        Sanctions. ANY USE OF THE WEBSITE NOT SPECIFICALLY PERMITTED UNDER THE TERMS OF USE IS STRICTLY PROHIBITED AND
+        MAY RESULT, AT SKYNET LABS’ DISCRETION, IN THE DELETION OF YOUR DATA AND/OR THE BLACKLISTING OF YOUR IP ADDRESS.
+      </P>
+      <P>
+        <strong>Contact.</strong> Any misuse of the Services may be reported to{" "}
+        <a className="text-primary" href="mailto:report@siasky.net">
```
Comment (peterjan) [inline packages/website/src/pages/terms.js:80]: this should be coming from the environment I think

File: packages/website/src/pages/terms.js [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+          report@siasky.net
```
Comment (peterjan) [inline packages/website/src/pages/terms.js:81]: same here

**Review by peterjan — COMMENTED** [review]
that was fast! thanks!

File: packages/website/src/pages/terms.js (hunk 1 above)
Comment (meeh0w) [reply packages/website/src/pages/terms.js:80]: @peterjan Yeah, these were hardcoded on the website recently (#1843), so I left them hardcoded here as well.

Are there env variables I can rely on for this?

**Review by peterjan — APPROVED** [review]

**Review by kwypchlo — APPROVED** [review]

**Pull request merged**